pages/contact.tsx

```diff
@@ -1,22 +1,54 @@
+import { EmailIcon, InfoIcon } from "@chakra-ui/icons";
 import {
-  Button,
   Flex,
+  Button,
   Heading,
   Text,
-  Link,
   Input,
   Textarea,
   FormControl,
   FormLabel,
+  InputLeftElement,
+  InputGroup,
+  Box,
 } from "@chakra-ui/react";
+import axios from "axios";
 import type { NextPage } from "next";
 import Head from "next/head";
+import { useForm } from "react-hook-form";
+
+const Contact: NextPage = () => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm();
 
-const Home: NextPage = () => {
+  const onSubmitForm = async (data: any) => {
+    const config = {
+      method: "POST",
+      url: "http://localhost:3000/api/hello",
```

Comment:
This url should work on all environments. Can we use the relative path here? `/api/hello`

Reply:
Additionally, could we not call postmark and handle the response here instead of calling the hello page?

Reply:
Thank you for the feedback. I will refactor the code and resubmit it for review very soon.